x-pack/plugins/cloud_integrations/cloud_links/public/maybe_add_cloud_links/appearance_selector/appearance_selector.test.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { coreMock } from '@kbn/core/public/mocks';
+import { securityMock } from '@kbn/security-plugin/public/mocks';
+
+import { AppearanceSelector } from './appearance_selector';
+
+describe('AppearanceSelector', () => {
```

Comment:
__issue:__ I observe some weird behavior in Serverless, did I do something wrong?

https://github.com/user-attachments/assets/8b7a2282-db3a-4c27-b1cf-c2760597429a

Reply:
Great catch, addressed in https://github.com/elastic/kibana/pull/203406/commits/c3144dc7e05677a6c8fc6c88a77a753e915b5fee